Pages/Signup/signup.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Veekay Accessories | Login</title>
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons+Outlined" rel="stylesheet">
+    <link rel="stylesheet" href="https://veekay-cmplib.netlify.app/CSS/components.css">
+    <link rel="stylesheet" href="https://veekay-cmplib.netlify.app/CSS/header.css">
+    <link rel="stylesheet" href="/CSS/header.css">
+    <link rel="stylesheet" href="signup.css">
+</head>
+</head>
+
+<body>
+    <div class="container">
+        <!-- Header Starts -->
+        <div class="header">
+            <div class="header_name">
+                <a href="/index.html"> <h3> Vee-Kay Accessories </h3></a>
+                <input class="input_box" type="text" placeholder="Enter text..">
+            </div>
+            <div class="header_subcontents">
+                <a href="../Login/login.html"><button class="btn btn-primary">Login</button></a>
+                <a href=""></a>
+                <div class="badge_container">
+                    <div class="icon_badge">
+                       <a href="../Cart Management/cart.html"> <span class="material-icons-outlined icon">
+                            shopping_cart
+                        </span></a>
+                        <span class="badge_numbered">1</span>
+                    </div>
+                    <div class="icon_badge">
+                       <a href="../Wishlist/wishlist.html"></a> <span class="material-icons-outlined icon">
+                            favorite_border
+                        </span>
+                        <span class="badge_numbered">1</span>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <!-- Header Ends -->
+
+        <div class="main_container">
+            <div class="form_container">
+                <form class="form">
+                    <div class="form_header">
+                        <h3>Signup</h3>
+                    </div>
+                    <div class="input_textbox ">
+                        <label class="form_label">Email Address:</label>
+                        <input class="input_box login_textbox" type="text" required
+                            placeholder="vikrant2812@gmail.com" />
+                    </div>
+                    <div class="input_textbox ">
+                        <label class="form_label">password:</label>
+                        <input class="input_box login_textbox" type="email" required placeholder="**********">
```

Comment:
check type attribute in both email and password.

Reply:
changed type="password"